ringbuf/reader_test.go

```diff
@@ -234,10 +240,68 @@ func TestReaderBlocking(t *testing.T) {
 	}
 }
 
+func TestReaderNoWakeup(t *testing.T) {
+	testutils.SkipOnOldKernel(t, "5.8", "BPF ring buffer")
+
+	prog, events := mustOutputSamplesProg(t,
+		sampleMessage{size: 5, flags: unix.BPF_RB_NO_WAKEUP}, // Read after timeout
+		sampleMessage{size: 6, flags: unix.BPF_RB_NO_WAKEUP}, // Discard
+		sampleMessage{size: 7, flags: unix.BPF_RB_NO_WAKEUP}) // Read won't block
```

Comment:
Nit: can you add `,` at the end?

Reply:
My gofmt removes it so I assume it might break the lint checks ?